firestore-counter/POSTINSTALL.md

```diff
@@ -18,17 +18,26 @@ match /databases/{database}/documents/pages/{page} {
 }
 ```
 
-#### Set up a scheduled function
 
-Review the [scheduled function documentation](https://firebase.google.com/docs/functions/schedule-functions) to set up a call to `${function:controller.url}` every minute. You may need to enable some APIs in your Firebase project to use scheduled functions.
+#### Set up a Cloud Scheduler job
 
-As an example, to set up a scheduled function, you can run the following [`gcloud`](https://cloud.google.com/sdk/gcloud/) commands:
+**IMPORTANT:** Note the following about v0.1.1 of this extension:
+- **If you updated your extension from v0.1.0 to v0.1.1:**  We recommend that you edit your Cloud Scheduler job to instead send a message to the extension's Pub/Sub topic, as described in this section. Although it's not recommended, if you leave your Cloud Scheduler job calling `${function:controller.url}`, your extension will continue to run as expected. For more information about the changes for v0.1.1, refer to the [changelog](https://github.com/firebase/extensions/blob/next/firestore-counter/CHANGELOG.md).
+- **If you installed this extension for the first time at v0.1.1:** Follow the instructions as described in this section.
+
+
+Set up a [Cloud Scheduler job](https://cloud.google.com/scheduler/docs/quickstart) to regularly send a message to the extension's Pub/Sub topic (`${param:EXT_INSTANCE_ID}`). This Pub/Sub topic then automatically triggers the controllerCore function (`${function:controllerCore}`). This controllerCore function is created by the extension and monitors the extension's workload.
```

Comment:
I'm not sure if "monitors the extension's workload" is the best way to describe it. It contains a part of the business logic of the extension, so it's more like it's carrying out the extension's workload rather than monitoring it.

Reply:
This is unchanged from the launch of the extension.  Is the functionality of this controllerCore function different from the functionality of the controller function?  We landed on this phrasing with @lepatryk .  Happy to update if y'all agree that it should be updated (we'd do it in several places throughout several files).

Reply:
I'm fine with changing the wording here to be more precise. In the description of the function, we say: "This function either aggregates shards itself, or it schedules and monitors workers to aggregate shards"

Reply:
I'm fine with using the same phrasing throughout each place this is mentioned. ok with y'all?

- extension.yaml, description of the function
- preinstall file, additional setup section
- postinstall file, cloud scheduler job section

Reply:
SGTM

Reply:
done :-D